core/src/main/scala/com/yahoo/maha/core/fact/Fact.scala

```diff
@@ -1556,6 +1560,7 @@ case class FactBuilder private[fact](private val baseFact: Fact, private var tab
       , revision
       , publicFact.dimRevision
       , Some(publicFact)
+      , publicFact.dimToRevisionMap ++ dimToRevisionOverrideMap
```

Comment:
Can you add test case to make that new values from dimToRevisionOverrideMap overwritten in the old map.